Vagrantfile

```diff
@@ -185,14 +185,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.proxy.enabled = { docker: false }
   end
 
+  etcd_seed_cluster = Net::HTTP.get(URI.parse("https://discovery.etcd.io/new?size=%d" % (NODES.to_i + 1)))
```

Comment:
I don't like the fact we are now adding a new dependency, namely the availability of this service.